hack/ci/e2e-k8s.sh

```diff
@@ -71,12 +71,12 @@ build_with_bazel() {
   bazel shutdown || true
   pkill ^bazel || true
 
-  # ensure the e2e script will find our binaries ...
-  # https://github.com/kubernetes/kubernetes/issues/68306
-  # TODO: remove this, it was fixed in 1.13+
-  mkdir -p '_output/bin/'
-  cp 'bazel-bin/test/e2e/e2e.test' '_output/bin/'
-  PATH="$(dirname "$(find "${PWD}/bazel-bin/" -name kubectl -type f)"):${PATH}"
+  local kubectl_path
+  # This is run from k/k root, so shellcheck shouldn't expect to be able to
+  # follow hack/lib/util.sh.
+  # shellcheck disable=SC1091
+  kubectl_path="$(source hack/lib/util.sh; kube::util::find-binary '.test')"
```

Comment:
we're going to be up a tree when we port this out of bash :|
https://github.com/kubernetes-sigs/kind/issues/986 https://github.com/kubernetes-sigs/kind/pull/1269